src/script/assets/AssetRepository.ts

```diff
@@ -146,11 +137,17 @@ export class AssetRepository {
   }
 
   private loadBuffer(asset: AssetRemoteData) {
-    const request = this.core.service!.asset.downloadAsset(asset.urlData, fraction => {
+    const isEncryptedAsset = !!asset.otrKey && !!asset.sha256;
+    const progressCallback = (fraction: number) => {
       asset.downloadProgress(fraction * 100);
-    });
-    asset.cancelDownload = request.cancel;
```

Comment:
Is this still being handled?

Reply:
Good catch, I think that's an oversight on my end 👁️

Reply:
@V-Gira fixed with https://github.com/wireapp/wire-webapp/pull/17164/commits/fd0a464096143027d3dcac91da5b4218e9a2885d

Reply:
nice :)